src/pages/Landing/index.tsx

```diff
@@ -45,111 +45,119 @@ export default () => {
     // TODO: Stretch - Use the options object to handle errors.
   );
 
-  const onSubmit = (e: FormEvent<Element>) => {
+  const onSubmit = async (e: FormEvent<Element>) => {
     e.preventDefault();
-    setNewUser({ isCool: false });
-    setIsModalOpen(false);
-  };
 
-  const columnHeaders = ['Name', 'Age', 'Is Cool'];
+    const apiUrl = 'http://localhost:3000/';
+    const response = await axios.post(apiUrl, { isCool: false });
```

Comment:
this doesn't need to call out to an actual api, just use the api 'wrapper' functions that provide a fake api:  https://github.com/content-services/React-typescript-demo/blob/main/src/api/CustomerApi.ts#L20-L30

You would call the addNewCustomer there and pass in the new customer that the modal is operating on